content/en/blog/2024/otel-collector-anti-patterns/index.md

```diff
@@ -75,7 +75,7 @@ before things start to escalate. This is where monitoring your Collectors can be
 very useful.
 
 But how does one monitor a Collector? The OTel Collector already emits
-[metrics for the purposes of its own monitoring](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/monitoring.md).
+[metrics for the purposes of its own monitoring](/docs/collector/internal-telemetry/#use-internal-telemetry-to-monitor-the-collector).
```

Comment:
The repo page was moved to otel.io, see:

- https://github.com/open-telemetry/opentelemetry-collector/pull/10454
- https://github.com/tiffany76/opentelemetry-collector/blob/ff305435928b90dade58ec38d6abb3d83c146262/docs/monitoring.md